src/llm/summarize/prompt.py

```diff
@@ -23,12 +23,39 @@
 [RESULT OF ACTION]
 {result}
 
-[INSTRUSCTION]
+[INSTRUCTION]
 Using above [THOUGHTS], [ACTION], and [RESULT OF ACTION], please summarize the event.
 
 [SUMMARY]
 """
 
+FINAL_ANSWER_TEMPLATE = """
+You are {name}, {role}
+You are working on {goal} and have completed all tasks but need find the final answer for the [GOAL].
```

Comment:
You are working on [GOAL] and have ...

(otherwise {goal} is mentioned twice, really close to each other